.github/workflows/test.yml

```diff
@@ -65,6 +65,7 @@ jobs:
           --no-healthcheck
           thunderstore:${GITHUB_SHA}
           --cov-report=xml:coverage_results/coverage.xml
+          --cov /app/
```

Comment:
Maybe we could still enable terminal cov report if you want to check it from the CI run logs?